### DISCUSSION TITLE
Change Loss Function of Loaded Model

### DISCUSSION THREAD

**Discussion by kamilm** [ThilinaRajapakse/simpletransformers#1555] [Q&A]
Hello, 

I enjoy the ease of use and quick iterations afforded by this package, so thanks creating and maintaining it. I have an obscure question: I would like to change the loss function of a loaded model, how can I do this? in particular, I would like to change tasks from classification to regression, so changing from cross entropy loss to mseloss. 

I don't know if what I want to achieve will work downstream, but this is my first hurdle.
